components/profile/create/stages/index.ts

```diff
@@ -0,0 +1,16 @@
+export { default as Introduction } from './Introduction.vue'
+export { default as Select } from './Select.vue'
+export { default as Form } from './Form.vue'
+export { default as Loading } from './Loading.vue'
+export { default as Success } from './Success.vue'
```

Comment:
how about use auto import instead? no need to set export and import manually

Reply:
was trying something else

not a fan of the long names that the auto import creates